File: Java Script - Introduction to Programming/9.9. Functions/04. Draw a Filled Square.js
function drawFilledSquare(n) {
    //top
    console.log('-'.repeat(n * 2))

    //middle
    for (let i = 0; i < n -2; i++) {
      console.log(`-${'\\/'.repeat(n - 1)}-`)
    }

    //bottom
    console.log('-'.repeat(n * 2))
}

// drawFilledSquare(4)
// drawFilledSquare(5)  

File: Pyrhon - Introduction to Programming/9.9. Functions/04. Draw a Filled Square.py
def draw_square(n):
    #top
    print('-'*(n * 2))

    #middle
    for i in range(n -2):
      print('-' + '\\/' * (n - 1) + '-')
    

    #bottom
    print('-' * (n * 2))


draw_square(int(input()))
